openedx/core/djangoapps/course_date_signals/handlers.py

```diff
@@ -177,7 +177,11 @@ def extract_dates(sender, course_key, **kwargs):  # pylint: disable=unused-argum
 
     date_items = extract_dates_from_course(course)
 
+    version = None
+    if hasattr(course, 'course_version'):
+        version = course.course_version
```

Comment:
Nit: I believe this could be rewritten as `version = getattr(course, 'course_version', None)`

Reply:
Done.